packages/bpk-component-calendar/src/BpkCalendarNav.tsx

```diff
@@ -127,10 +131,16 @@ const BpkCalendarNav = ({
               disabled || !isWithinRange(prevMonth, { start: min, end: max })
             }
           >
-            <ArrowLeftIcon 
-            // TODO: className to be removed
-            // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-            className={getClassName('bpk-calendar-nav__icon')} />
+            <ArrowLeftIcon
+              // TODO: className to be removed
+              // eslint-disable-next-line @skyscanner/rules/forbid-component-props
```

Comment:
Remaining to solve in the next ticket, required to set RTL support on the icons.